spanner/spanner_samples.rb

```diff
@@ -2005,6 +2052,50 @@ def create_backup_with_encryption_key project_id:, instance_id:, database_id:, b
   # [END spanner_create_backup_with_encryption_key]
 end
 
+def create_backup_with_multiple_kms_keys project_id:, instance_id:, database_id:, backup_id:, kms_key_names:
+  # [START spanner_create_backup_with_MR_CMEK]
+  # project_id  = "Your Google Cloud project ID"
+  # instance_id = "Your Spanner instance ID"
+  # database_id = "Your Spanner database ID"
+  # backup_id = "Your Spanner backup ID"
+  # kms_key_names = "Your list of backup encryption database KMS keys"
+
+  require "google/cloud/spanner"
+  require "google/cloud/spanner/admin/database"
+
+  database_admin_client = Google::Cloud::Spanner::Admin::Database.database_admin
+
+  instance_path = database_admin_client.instance_path project: project_id, instance: instance_id
+  db_path = database_admin_client.database_path project: project_id,
+                                                instance: instance_id,
+                                                database: database_id
+  backup_path = database_admin_client.backup_path project: project_id,
+                                                  instance: instance_id,
+                                                  backup: backup_id
+  expire_time = Time.now + (14 * 24 * 3600) # 14 days from now
+  encryption_config = {
+    encryption_type: :CUSTOMER_MANAGED_ENCRYPTION,
+    kms_key_names:    kms_key_names
+  }
+
+  job = database_admin_client.create_backup parent: instance_path,
+                                            backup_id: backup_id,
+                                            backup: {
+                                              database: db_path,
+                                                expire_time: expire_time
+                                            },
+                                            encryption_config: encryption_config
```

Comment:
Is there a reason you're setting the encryption config as a temporary local variable, but constructing the backup config inline? Can you choose one style or the other so the sample looks consistent?